docs/preview/features/powershell/azure-api-management.md

```diff
@@ -45,16 +45,32 @@ PS> Create-AzApiManagementApiOperation -ResourceGroup $ResourceGroup -ServiceNam
 # New API operation '$OperationName' on API Management service was added.
 ```	
 
-## Import policy to an API in the API Management service
+## Remove all Azure API Management defaults from the service
+
+Remove all default API's and products from the Azure API Management service ('echo-api' API, 'starter' & 'unlimited' products), including the subscriptions.
+
+| Parameter       | Mandatory | Description                                              |
+| --------------- | --------- | -------------------------------------------------------- |
+| `ResourceGroup` | yes       | The resource group containing the Azure API Management service |
+| `ServiceName`   | yes       | The name of the Azure API Management service located in Azure  |
```

Comment:
Reason why I'm renaming all of them is that we are referring to the instance of the service, not the service itself #nitpicking
```suggestion
| `ServiceName`   | yes       | The name of the Azure API Management instance located in Azure  |
```

Reply:
This makes me wonder, if all open APIM PRs are merged if we should rename all `ServiceName` parameters to `InstanceName`; but just a suggestion. Can be confusing as well if it's multi-instance setup.

Up to @mbraekman

Reply:
Good point, agree with this one.
#62 created for this.